### PR TITLE
Upgrade to Vertexium 3.1.0

### PR DESCRIPTION
**Pull request by joeferner**
- [x] joeferner
- [x] mwizeman joeybrk372 jharwig sfeng88

Points of Regression:
- Be sure to install root/pom.xml
- Big change comes with aggregation on exact matches and search of exact matches with case insensitive searching.
- See https://github.com/visallo/vertexium/releases/tag/vertexium-root-3.1.0 for other changes 

CHANGELOG
Changed: Upgrade to Vertexium 3.1.0

### CODE REVIEW

File: core/core/src/main/java/org/visallo/core/cache/CacheOptions.java [hunk 1]
```diff
@@ -1,7 +1,11 @@
 package org.visallo.core.cache;
 
+import java.util.concurrent.TimeUnit;
+
 public class CacheOptions {
     private Long maximumSize;
+    private Long expireAfterAccessDuration;
```
Comment (mwizeman) [inline core/core/src/main/java/org/visallo/core/cache/CacheOptions.java]: Did you mean for this change to be part of this PR?

Comment (joeferner) [reply core/core/src/main/java/org/visallo/core/cache/CacheOptions.java]: No, those must have made their way in when I was trying to fix the ontology cache. I'll remove them tonight.

**Comment by joeferner** [comment]
Needed to make the following change to Vertexium... https://github.com/visallo/vertexium/commit/64db7c34e686ac72c6730489fc093dbbdf12d1aa

**Comment by joeybrk372** [comment]
LGTM

**Pull request merged**